pom.xml

```diff
@@ -193,6 +195,17 @@
                 <artifactId>mockito-core</artifactId>
                 <version>${mockito-core.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.codehaus.groovy</groupId>
+                <artifactId>groovy-all</artifactId>
+                <version>${groovy.version}</version>
+                <type>pom</type>
+            </dependency>
+            <dependency>
```

Comment:
❓ Why is this here a second time and without the `type` qualifier? Are we pulling in both the pom and a jar?

Reply:
The first one is the standard way to get dependencies defined in the pom file. The second is because this version of `groovy-all` also contains classes. The latest one used for Java 11 is a pure pom and as such, the second one should go away.